### ISSUE TITLE
Failed to compile bacon 2.2.0

### ISSUE THREAD

**Issue by ghost** [Canop/bacon#82]
I'm on Void Linux x86_64, kernel 5.16.20

rustc 1.61.0

```
Compiling bacon v2.2.0
error[E0308]: mismatched types
  --> /home/crdpa/.cargo/registry/src/github.com-1ecc6299db9ec823/bacon-2.2.0/src/app.rs:47:21
   |
46 |                 match user_event?.event {
   |                       ----------------- this expression has type `crokey::__private::crossterm::event::Event`
47 |                     Event::Resize(mut width, mut height) => {
   |                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected enum `crokey::__private::crossterm::event::Event`, found enum `crossterm::event::Event`
   |
   = note: perhaps two different versions of crate `crossterm` are being used?

error[E0308]: mismatched types
  --> /home/crdpa/.cargo/registry/src/github.com-1ecc6299db9ec823/bacon-2.2.0/src/app.rs:57:21
   |
46 |                 match user_event?.event {
   |                       ----------------- this expression has type `crokey::__private::crossterm::event::Event`
...
57 |                     Event::Key(key_event) => {
   |                     ^^^^^^^^^^^^^^^^^^^^^ expected enum `crokey::__private::crossterm::event::Event`, found enum `crossterm::event::Event`
   |
   = note: perhaps two different versions of crate `crossterm` are being used?

error[E0277]: the trait bound `crokey::CroKey: std::convert::From<crossterm::event::KeyEvent>` is not satisfied
  --> /home/crdpa/.cargo/registry/src/github.com-1ecc6299db9ec823/bacon-2.2.0/src/app.rs:58:51
   |
58 |                         debug!("key pressed: {}", CroKey::from(key_event));
   |                                                   ^^^^^^^^^^^^ the trait `std::convert::From<crossterm::event::KeyEvent>` is not implemented for `crokey::CroKey`
   |
   = help: the following implementations were found:
             <crokey::CroKey as std::convert::From<crokey::__private::crossterm::event::KeyEvent>>

error[E0277]: the trait bound `crokey::CroKey: std::convert::From<crossterm::event::KeyEvent>` is not satisfied
  --> /home/crdpa/.cargo/registry/src/github.com-1ecc6299db9ec823/bacon-2.2.0/src/app.rs:59:50
   |
59 |                         action = keybindings.get(key_event);
   |                                              --- ^^^^^^^^^ the trait `std::convert::From<crossterm::event::KeyEvent>` is not implemented for `crokey::CroKey`
   |                                              |
   |                                              required by a bound introduced by this call
   |
   = help: the following implementations were found:
             <crokey::CroKey as std::convert::From<crokey::__private::crossterm::event::KeyEvent>>
   = note: required because of the requirements on the impl of `Into<crokey::CroKey>` for `crossterm::event::KeyEvent`
note: required by a bound in `keybindings::KeyBindings::get`
  --> /home/crdpa/.cargo/registry/src/github.com-1ecc6299db9ec823/bacon-2.2.0/src/keybindings.rs:58:20
   |
58 |     pub fn get<CK: Into<CroKey>>(&self, key: CK) -> Option<&Action> {
   |                    ^^^^^^^^^^^^ required by this bound in `keybindings::KeyBindings::get`

error[E0308]: mismatched types
  --> /home/crdpa/.cargo/registry/src/github.com-1ecc6299db9ec823/bacon-2.2.0/src/help_page.rs:45:47
   |
45 |         skin.italic = CompoundStyle::new(Some(AnsiValue(204)), None, Attribute::Bold.into());
   |                                               ^^^^^^^^^^^^^^ expected enum `crokey::__private::crossterm::style::Color`, found enum `crossterm::style::Color`
   |
   = note: perhaps two different versions of crate `crossterm` are being used?

error[E0277]: the trait bound `crokey::__private::crossterm::style::Attributes: std::convert::From<crossterm::style::Attribute>` is not satisfied
  --> /home/crdpa/.cargo/registry/src/github.com-1ecc6299db9ec823/bacon-2.2.0/src/help_page.rs:45:86
   |
45 |         skin.italic = CompoundStyle::new(Some(AnsiValue(204)), None, Attribute::Bold.into());
   |                                                                                      ^^^^ the trait `std::convert::From<crossterm::style::Attribute>` is not implemented for `crokey::__private::crossterm::style::Attributes`
   |
   = help: the following implementations were found:
             <crokey::__private::crossterm::style::Attributes as std::convert::From<&[crokey::__private::crossterm::style::Attribute]>>
             <crokey::__private::crossterm::style::Attributes as std::convert::From<crokey::__private::crossterm::style::Attribute>>
   = note: required because of the requirements on the impl of `Into<crokey::__private::crossterm::style::Attributes>` for `crossterm::style::Attribute`

error[E0308]: mismatched types
  --> /home/crdpa/.cargo/registry/src/github.com-1ecc6299db9ec823/bacon-2.2.0/src/state.rs:62:23
   |
62 |             .set_fgbg(AnsiValue(252), AnsiValue(239));
   |                       ^^^^^^^^^^^^^^ expected enum `crokey::__private::crossterm::style::Color`, found enum `crossterm::style::Color`
   |
   = note: perhaps two different versions of crate `crossterm` are being used?

error[E0308]: mismatched types
  --> /home/crdpa/.cargo/registry/src/github.com-1ecc6299db9ec823/bacon-2.2.0/src/state.rs:62:39
   |
62 |             .set_fgbg(AnsiValue(252), AnsiValue(239));
   |                                       ^^^^^^^^^^^^^^ expected enum `crokey::__private::crossterm::style::Color`, found enum `crossterm::style::Color`
   |
   = note: perhaps two different versions of crate `crossterm` are being used?

error[E0308]: mismatched types
  --> /home/crdpa/.cargo/registry/src/github.com-1ecc6299db9ec823/bacon-2.2.0/src/state.rs:63:54
   |
63 |         status_skin.italic = CompoundStyle::new(Some(AnsiValue(204)), None, Attribute::Bold.into());
   |                                                      ^^^^^^^^^^^^^^ expected enum `crokey::__private::crossterm::style::Color`, found enum `crossterm::style::Color`
   |
   = note: perhaps two different versions of crate `crossterm` are being used?

error[E0277]: the trait bound `crokey::__private::crossterm::style::Attributes: std::convert::From<crossterm::style::Attribute>` is not satisfied
  --> /home/crdpa/.cargo/registry/src/github.com-1ecc6299db9ec823/bacon-2.2.0/src/state.rs:63:93
   |
63 |         status_skin.italic = CompoundStyle::new(Some(AnsiValue(204)), None, Attribute::Bold.into());
   |                                                                                             ^^^^ the trait `std::convert::From<crossterm::style::Attribute>` is not implemented for `crokey::__private::crossterm::style::Attributes`
   |
   = help: the following implementations were found:
             <crokey::__private::crossterm::style::Attributes as std::convert::From<&[crokey::__private::crossterm::style::Attribute]>>
             <crokey::__private::crossterm::style::Attributes as std::convert::From<crokey::__private::crossterm::style::Attribute>>
   = note: required because of the requirements on the impl of `Into<crokey::__private::crossterm::style::Attributes>` for `crossterm::style::Attribute`

Some errors have detailed explanations: E0277, E0308.
For more information about an error, try `rustc --explain E0277`.
error: could not compile `bacon` due to 10 previous errors
error: failed to compile `bacon v2.2.0`, intermediate artifacts can be found at `/tmp/cargo-installfx1qJp`
⏎                                                                              
```

**Comment by Canop** [comment]
Damn...

I'll fix that ASAP.

**Comment by Canop** [comment]
Can you please confirm that `cargo install --locked bacon` does work ?

**Comment by crdpa** [comment]
> Can you please confirm that `cargo install --locked bacon` does work ?

It is working now.

**Comment by Canop** [comment]
I've also released bacon v2.2.1 which doesn't need the `--locked`.
